pykickstart/commands/firewall.py

```diff
@@ -218,14 +218,23 @@ class F20_Firewall(F14_Firewall):
     def __init__(self, writePriority=0, *args, **kwargs):
         F14_Firewall.__init__(self, writePriority, *args, **kwargs)
         self.remove_services = kwargs.get("remove_services", [])
+        self.use_system_defaults = kwargs.get("use_system_defaults", None)
 
     def _getParser(self):
         op = F14_Firewall._getParser(self)
         op.add_argument("--remove-service", dest="remove_services", help="",
                         action=ExtendAction, type=commaSplit, version=F20)
+        op.add_argument("--use-system-defaults", dest="use_system_defaults",
+                        version=F20, help="""
```

Comment:
Shouldn't this have a higher `version=`?

Reply:
I was mostly matching the `--remove-service` option that was already there. Depending on this review I might have to make a whole new subclass for F28, but I don't know what the convention is.

Reply:
This is probably a little confusing and could be better documented... somewhere.  In this context, the version= parameter is used to generate documentation.  For your new option, you would use version=F28 (or whatever).

The general idea is that the version is whatever release of Fedora or RHEL it first appeared in.  You would want to make a new subclass here, yes.  I'd be happy to help with that if you need pointers.

Reply:
I can take a stab at it. would I have to create a new subclass for the other branches as well?